.buildkite/pipeline.yml

```diff
@@ -20,7 +20,7 @@ steps:
     command: ".buildkite/commands/installable-build-wordpress.sh"
     env: *common_env
     plugins: *common_plugins
-    if: build.pull_request.id != null
+    if: "build.pull_request.id != null || build.pull_request.draft"
```

Comment:
Just curious: Did this fail without `""` or did you add them for style/other reason?

Context. I don't know why, but I have a weird reaction to seeing quotes when they're not required 🙃 Maybe it's working with Ruby and all the `'` vs `"` Rubocop warnings.

Reply:
> might not be needed but better safe than sorry?

👍 👍 👍